### DISCUSSION TITLE
KATA 3 velociada del asteroide

### DISCUSSION THREAD

**Discussion by AlejandroNav** [LaunchX-InnovaccionVirtual/CursoIntroPython#53] [Q&A] [answered]
Hola, tengo una diminuta duda sobre el final del KATA 3, donde se menciona que un asteroide causa mucho dano mientras su tamano sea mayor a 25 pero menor a 1000, pero luego no se vuelve a mencionar nada sobre el limite superior del tamano. Mi codigo ahsta el momento es este:

velo = 78
tam = 345
```

if velo  > 25 and 1000> tam  > 25 :
    print("Alerta maxima")
elif velo  >= 20:
    print(' Busca la luz')
elif tam  < 25:
    print('sin peligro ')
else:
    print('sin peligro ')
```

Creo que este codigo satisface toda la informacion que aparece en el Kata, pero siento que falta algo relacionado a lo de la velociadad superior a 999, quizas un if anidado donde si el asteroide lleva una velocidad mayor a los 25 km/s y un tamano superior a 999 metros se de una alerta de un peligro mayor, solo queria estar seguro de que no me falte informacion, quizas lo estoy sobrepensando.

**Comment by PlayerOneV** [comment, marked as answer]
Yo no tome en cuenta el limite del tamaño de 1000 por que el objetivo del las condicionales es advertir sobre el peligro y para que un asteroide no se deshaga en la atmosfera debe tener un tamaño mayor a 25 así que cualquier asteroide mas grande que eso necesita una advertencia.
Si gustas puedes poner mas condicionales y mostrar una advertencia por si el asteroide llega a un tamaño de 1000m o superior.

**Comment by khisus19** [comment]
Este ejercicio también me dio problemas. Siento que está mal planteado. ¿Qué sucedería si un asteroide de más de 1000 metros viene a menos de 20 km/s en dirección a la tierra? Igualmente causaría mucho daño. Maybe I am over thinking this.

**Reply by AzaelBarbosa** [reply]
Yo si puse esta condicional ya que creo que no importa la velocidad si es de mas de 1000 metros no se desintegra y causa daño jeje

**Reply by GloriaCoralCerecedo** [reply]
Igual a mi me confundió mucho. Pero ya entendí es que dice que si el tamaño más grande 25 pero menor que 1000 causara daño, entonces supongo que los que son más de 1000 no causara daño, según lo leído, solo importa ese rango o más fácil que simplemente sea mayor a 25. Igual con  la velocidad solo dice que hay peligro con mayor a 25, no menos.

**Reply by khisus19** [reply]
> Igual a mi me confundió mucho. Pero ya entendí es que dice que si el tamaño más grande 25 pero menor que 1000 causara daño, entonces supongo que los que son más de 1000 no causara daño, según lo leído, solo importa ese rango o más fácil mayor a 25. Igual con  la velocidad solo dice que hay peligro con mayor a 25, no menos.  

Pero eso carece de lógica en la realidad. ¿Cómo un asteroide de más de 1000 metros no causaría daño y uno de 999 sí? Si es un asteroide del tamaño de la luna impacta con la tierra nos hará polvo cósmico sin importar si viene a la velocidad de un caracol.

**Reply by AzaelBarbosa** [reply]
Creo yo que fue error de redacción.

**Reply by GloriaCoralCerecedo** [reply]
Si opino igual que si es más grande puede causar daño, pero yo me refiero a que lo que comente anteriormente es lo que entendí con el ejercicio. Igual se puede poner una condicional

**Reply by carlos-gmx3** [reply]
Yo si puse un caso para  asteroides con un tamaño mayor a 1000m solo para probar jeje
Le advertía al usuario que el asteroide podría tener el tamaño como para extinguir todo en un área considerable, que revise su trayectoría
